src/domain/event/EditEventPage.tsx

```diff
@@ -135,6 +138,7 @@ const EditEventPage: React.FC = () => {
       goToEventDetailsPage();
     } catch (e) {
       if (process.env.NODE_ENV === 'test') {
```

Comment:
Maybe helper function to test is it testing environment

Reply:
fixed